OWNERS.md

```diff
@@ -120,7 +120,7 @@ Scope: Tooling for frontend and backend schema-first OpenAPI development.
 
 | Name                           | Organization              | GitHub                                                | Discord                        |
 | ------------------------------ | ------------------------- | ----------------------------------------------------- | ------------------------------ |
-| Adam Harvey                    | Cisco                     | [adamdmharvey](https://github.com/adamdmharvey)       | `adamharvey#3739`              |
+| Adam Harvey                    | Cisco                     | [adamdmharvey](https://github.com/adamdmharvey)       | `adamharvey_`                  |
 | Alex Crome                     |                           | [afscrome](https://github.com/afscrome)               | `afscrome`                     |
 | Andre Wanlin                   | Keyloop                   | [awanlin](https://github.com/awanlin)                 | `Ahhhndre#3095`                |
```

Comment:
Would you be open to updating mine while you are at it? New handle is just `ahhhndre`

Reply:
@awanlin ; sure.  fire a suggestion and i'll commit it, so it'll give you credit on the commit as a co-author!